### PR TITLE
docs: Clarify how panic mode works with priorities

### PR DESCRIPTION
**Pull request by snowp**
*title*: docs: Clarify how panic mode works with priorities

*Description*:
Clarifies that panic mode counts the number of healthy nodes per priority
and triggers panic mode for a priority. Based on my reading of this code: https://github.com/envoyproxy/envoy/blob/master/source/common/upstream/thread_aware_lb_impl.cc#L24-L30

*Risk Level*: 
Low: Documentation change

*Testing*:
N/A

*Docs Changes*:
See summary

*Release Notes*:
N/A

### CODE REVIEW

File: docs/root/intro/arch_overview/load_balancing.rst [hunk 1]
```diff
@@ -124,6 +124,9 @@ runtime as well as in the :ref:`cluster configuration
 is used to avoid a situation in which host failures cascade throughout the cluster as load
 increases.
 
+Note that panic thresholds are *per-priority*. This means that if the percentage of healthy nodes
+in a single priority goes below the threshold, that priority is will enter panic mode.
```
Comment (zuercher) [inline docs/root/intro/arch_overview/load_balancing.rst]: nit: "that priority will enter panic mode" (remove extra is)

**Comment by snowp** [comment]
@alyssawilk I added in a line about discouraging the use of both at the same time like you suggested. Seems like a good recommendation to keep in the docs (we ended up disabling panic thresholds ourselves).

**Comment by alyssawilk** [comment]
Alternately we could rework how the two interact so when using failover, panic thresholds only apply if there's sufficient upstreams at any priority level to serve.    So if P=0 is below panic threshhold we simply spill over, but it P=0-P=n are all down, we finally apply it.

**Comment by mattklein123** [comment]
> Alternately we could rework how the two interact so when using failover, panic thresholds only apply if there's sufficient upstreams at any priority level to serve. So if P=0 is below panic threshhold we simply spill over, but it P=0-P=n are all down, we finally apply it.

Yeah that makes sense also. Let me open a ticket to track.

**Pull request merged**